man/render.Rd

```diff
@@ -117,49 +124,46 @@ is passed to \code{render} then the first one defined will be used:
 \preformatted{
 output:
   pdf_document:
-    toc: true
-    highlight: zenburn
-  html_document:
-    toc: true
-    theme: united
+  toc: true
+highlight: zenburn
+html_document:
+  toc: true
+theme: united
```

Comment:
Same problem here.

Reply:
I can fix these. Thought the formatting survived the copy/paste but no.

Reply:
Please feel free to push to master for these two changes. Thanks!

Reply:
Just did this. Thanks for reviewing!